corehq/form_processor/management/commands/set_meta_fields.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import datetime
+
+from bulk_update.helper import bulk_update as bulk_update_helper
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from corehq.form_processor.models import XFormInstanceSQL
+from corehq.sql_db.util import split_list_by_db_partition
+from dimagi.utils.chunked import chunked
+
+
+logger = logging.getLogger('set_meta_fields')
+logger.setLevel('DEBUG')
+
+META_FIELDS = ("time_end", "time_start", "commcare_version", "build_version")
+
+
+def form_ids_by_domain_received_on(domain, from_date, to_date):
+    from corehq.sql_db.util import run_query_across_partitioned_databases
+    q_expr = Q(domain=domain, received_on__gt=from_date, received_on__lt=to_date)
```

Comment:
@snopoke should we just remove the domain filter and do this for just based on received_on?

Reply:
seems reasonable